src/util/api/api.js

```diff
@@ -17,6 +17,8 @@ function getLikeCoinButtonHeaders({
 
 export const apiGetUserMinById = id => axios.get(`${LIKECOIN_API}/users/id/${id}/min`);
 
+export const apiGetURLToISCNMapping = (likerId, referrer) => axios.get(`${LIKECOIN_API}/like/iscn/mapping?liker=${encodeURIComponent(likerId)}&url=${encodeURIComponent(referrer)}`);
```

Comment:
https://github.com/likecoin/likecoin-api-private/pull/352#:~:text=Also%20liker%20should,1
need change to likerId

Reply:
for qs please do underscore, i.e. `liker_id`

Reply:
ok, fix to liker_id ok in private api

Reply:
help me change it to liker_id here, thx